index.html

```diff
@@ -752,6 +752,18 @@ <h3>Validate Algorithm</h3>
 as indicated in the `statusMessages` array.
         </li>
       </ol>
+
+      <p>
+When a `statusListCredential` URL is dereferenced, server implementations MAY provide a
+mechanism to dereference the status list as of a particular point in time. If such a
+feature is supported, and if query parameters are supported by the URL scheme,
+then the name of the query parameter MUST be `timestamp` and the value MUST be
+a valid URL-encoded [[XMLSCHEMA11-2]] dateTimeStamp string value. The result of
+dereferencing such a timestamped URL MUST be a status list credential containing
+either the status list as it existed at the given point in time, or an error.
+point in time.
```

Comment:
```suggestion
dereferencing such a timestamped URL MUST be a status list credential containing
either the status list as it existed at the given point in time, or a 
<a href="#STATUS_RETRIEVAL_ERROR">STATUS_RETRIEVAL_ERROR</a>.
```

Reply:
Before the above is applied... should a STATUS_RETRIEVAL_ERROR or other error be returned in a (status list) [verifiable] credential, or raw? This may change where the "either" belongs.